docs/satellites.md

```diff
@@ -0,0 +1,248 @@
+  comparing baseline data with the actual status and drawing conclusions
+  on the impact data for many sectors and types of projects. Depending
+  on project type, they can also be used for progress monitoring. Since
+  they do not require intermediaries, they are often used for remote
+  verification, but they can be used by all stakeholders for decision making along the project cycle and
+  during every step in the project cycle (e.g., planning and appraisal,
+  implementation, progress monitoring and verification, impact
+  monitoring, and evaluation). Some examples:
+
+-   **Feature mapping** (e.g., buildings, roads, landcover, and water extent)
+
+-   **Change Detection** (e.g., urban growth, deforestation, agricultural expansion, bio-diversity monitoring, climate vulnerability,
+    and glacier melting)
+
+-   **Elevation or Digital Elevation Models** (DEM) (e.g., 3D terrain models for flood risk and 3D urban models)
+
+-   **Monitoring in fragile or disaster contexts** (refugee routes,
+    drought and floods, locusts, etc.)
+
+### Step 3: What Should I Consider When Acquiring Satellite Imagery?
+ 
+  First, define and formulate your information need. Your need (e.g.,
+  detecting deforestation in the Amazon) will direct you to choose the
+  type of satellite image suitable for your case and analysis. Your
+  options:
+
+-   **Open and free-access satellite data:** this includes most of
+    optical and radar satellites from public-funded satellite missions
+    from the EU and the US (such as, Landsat, Sentinel or  ASTER). These data are available with high and low resolutions under
+  free and open data policies and are accessible through several
+  platforms (e.g., through the USGS, the Landsat mission, and Copernicus
+  Access Hubs) (see [Links below](#links-to-further-sources)). The [Fact Sheets Geospatial Tools and Data Sources](gis.md) provide
+further information regarding free and open access to satel-
+lite imagery. KfW’s evaluation department has initiated a
+search engine for open geodata sets called [mapme](https://mapme.
+shinyapps.io/geodata)
+
+-   **Commercial satellite data** (privately funded or sometimes
+    publicly funded missions accessible for free under restricted terms
+    (proposals, project context, research) or against a commercial
+    fee): If the imagery requirements are much higher (e.g., very high
+    resolution, near-real time, tight observation windows), the image
+    data products will need to be purchased under commercial licenses
+    from EO industry platforms directly (see [Links below](#links-to-further-sources)) or using image broker services.
+
+-   **Satellite Image data** from public or commercial sources **can be
+    procured at various processing levels**, also saving several
+    preprocessing steps and thus time and cost (the pre- processing levels include radiometric, atmospheric, geometric
+  corrections and orthorectification using DEM). This will ensure the
```

Comment:
please insert spaces from line 82 to line 84 to have the formatting coherent with the bullet points above